.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ commands:
     - run:
         name: Validate orbs
         command: |
+            export GITHUB_TOKEN=$GH_TOKEN
```

Comment:
I created a GITHUB_TOKEN env variable (copy of GH_TOKEN) in helix-continuous.
Just commenting, do not think this is so much better. At least there is central place of all env var (even copies with different names), instead of having the code to manage the env var names...

Reply:
Good point. I think we should eventually add it to the Project Helix context in CircleCI...